pyproject.toml

```
@@ -1,137 +1,164 @@
-[tool.poetry]
+[project]
 name = "pymedphys"
 version = "0.42.0-dev0"
-readme = "README.rst"
 description = "Medical Physics library"
-authors = ["PyMedPhys Contributors <developers@pymedphys.com>"]
-license = "Apache-2.0"
-homepage = "https://pymedphys.com"
-repository = "https://github.com/pymedphys/pymedphys"
-documentation = "https://docs.pymedphys.com"
+readme = "README.rst"
+requires-python = ">=3.10,<3.13"
```

One of the benefits of uv over poetry is that we no longer are forced to do upper pins here.  Upper pins often end up causing tangles unless they are absolutely known to be needed.  

```suggestion
requires-python = ">=3.10"
```

---

Thanks @SimonBiggs

The CI was running into issues with python 3.13. I was thinking of submitted a follow up PR explicitly supporting python 3.13 - is that okay?